src/oneshot/posix.rs

```diff
@@ -70,16 +77,15 @@ fn time_create(state: *mut TimerState) -> RawFd {
     event.sigev_signo = TIMER_SIG;
     event.sigev_notify = libc::SIGEV_SIGNAL;
 
-    let mut res = 0;
+    let mut res = mem::MaybeUninit::<ffi::timer_t>::uninit();
```

Comment:
MaybeUninit is redandant for integers (there can be no Drop)

Reply:
`timer_t` is no longer an int, technically it's a pointer. It can either be zeroed or left uninitialized like this. I chose the latter because it will be initialized by `create_timer`.

Reply:
yeah I know, I mean you could assume_init right here.

Anyway, I don't mind this change, but I don't think I'd go for changing usize to custom timer_t.
(my policy is less code is better, even if it is boilerplate)

Reply:
Hm, yeah probably. Whether uninitialized raw pointers are UB or not is still an [open discussion](https://github.com/rust-lang/unsafe-code-guidelines/issues/71), but it seems to lean towards "no". Until it's been properly established, doing it like this is the more conservative option.